utils/bin/launch.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+source /opt/run-java/dynamic_resources.sh
```

Comment:
This path should be changed as well. Have a look at latest version in the EnMasse master branch

Reply:
Ditto above, thanks.